themes/default/content/blog/introducing-azure-native-v2/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Introducing Azure Native 2.0"
+
+date: 2023-06-13T08:00:00-04:00
+
+meta_desc: "Introducing Pulumi Azure Native Provider 2.0: Streamlined, Expanded, and More Powerful than Ever!"
+
+meta_image: meta.png
+
+authors:
+    - monica-rodriguez
+
+tags:
+    - providers
+    - azure
+
+---
+
+{{% notes type="warning" %}}
+The Pulumi Azure Native Provider 2.0 is currently available in beta.
+{{% /notes %}}
+
+We are thrilled to announce the highly anticipated beta release of the Pulumi Azure Native Provider 2.0, a significant upgrade to our infrastructure as code (IaC) solution for Microsoft Azure. This release brings a host of exciting features and improvements that will enhance your experience with managing Azure resources and empower you to build robust and scalable cloud infrastructure more efficiently.
+
+<!--more-->
+
+At Pulumi, we understand the importance of keeping up with the ever-evolving cloud landscape. The Pulumi Azure Native Provider 2.0 represents our commitment to providing you with the best tools to harness the full power of Microsoft Azure while providing you with same-day access to the entire [Azure API surface](https://docs.microsoft.com/en-us/rest/api/azure/). Let's dive into some of the key enhancements in this release:
+
+## Reduced SDK Size for Faster Deployments
+
+We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a remarkable decrease in SDK size of up to 60% <!-- TODO: Get latest size updates on alpha.3 once released -->. This improvement translates into faster deployment times, reduced resource consumption, and an overall smoother experience when working with the provider.
```

Comment:
I would expect deployment times to stay pretty much the same, no? Unless you're memory-constrained. I think the speed-up at development time and build time is more significant.

Reply:
Doesn't deploy time include build time?

Reply:
Deployment could be faster in CI, as we've heard from feedback. I think development time is a better call out.